ui/src/components/device/DeviceAdd.vue

```diff
@@ -76,6 +77,14 @@
 export default {
   name: 'DeviceAdd',
 
+  props: {
```

Comment:
Is this `smallButton` props really needed? Couldn't you just use `true` in `:small`?

Reply:
Good point.

Reply:
@gustavosbarreto @leonardojoao this comment from @noreng-jg has not been resolved.